assets/js/base/context/cart-checkout/checkout-state/index.js

```diff
@@ -173,7 +173,7 @@ export const CheckoutStateProvider = ( {
 
 	// emit events.
 	useEffect( () => {
-		const { status } = checkoutState;
+		const status = checkoutState.status;
```

Comment:
I had to make this change because the hook linter doesn't pick up the destructuring on the object so thinks `checkoutState` needs to be a dependency. I hate making changes like this just to satisfy linting, but it's not a super big deal in this case.